model/model_training/custom_datasets/qa_datasets.py

```diff
@@ -552,6 +552,7 @@ class DatabricksDolly15k(Dataset):
     def __init__(self, cache_dir: str | Path, mode: str = "sft", input_max_length: int = 2048) -> None:
         super().__init__()
         self.rows = []
+        self.citation_regex = re.compile(r"\[[a-z]\]")  # removes citations in the form of e.g. [a]
```

Comment:
could we please also use the same regex that is defined here: https://github.com/LAION-AI/Open-Assistant/blob/2e6f22c2a137ee60ef68cf908073c1e29babd633/model/model_training/custom_datasets/qa_datasets.py#L20

maybe also add upper case single latter references?

Reply:
we use that regex already, see line 565. Will add upper case letters aswell